src/utils/decorators/input-label/input-label.js

```diff
@@ -67,7 +67,7 @@ const InputLabel = (ComposedComponent) => {
        * Pass true to format the input/label inline
        *
        * @property
-       * @default top
+       * @default false
```

Comment:
I'm not sure these definitions (`@property`, `@default`, `@type`) are being used anymore, so can probably remove.

Reply:
you are right, thanks